openedx_events/event_bus/tests/test_loader.py

```diff
@@ -112,3 +112,17 @@ def test_default_does_nothing(self):
                 event_key_field='user.id', event_data={},
                 event_metadata=EventsMetadata(event_type='eh')
             ) is None
+
+
+class TestConsumer(TestCase):
+
+    @override_settings(EVENT_BUS_CONSUMER=None)
+    def test_default_does_nothing(self):
```

Comment:
is there a way to mock the not default behavior?

Reply:
@mariajgrimaldi The loader is already tested [above](https://github.com/openedx/openedx-events/pull/200/files#diff-d3aa2802dab4bb0b7464d190bc4f0c0b07655847877298bd51ea6ada75051a0fR39-R45) and we don't need to test `consume_indefinitely` here.

Reply:
good! thanks